frontend/src/message.d.ts

```diff
@@ -3,6 +3,7 @@ type Message = {
   content: string;
   imageUrls: string[];
   timestamp: string;
+  pending: boolean;
```

Comment:
might just want to make this optional so we don't have to explicitly say `pending: false` everywhere
```suggestion
  pending?: boolean;
```